TranslatorReasonerAPI.yaml

```diff
@@ -245,13 +251,19 @@ components:
         - id
     EdgeBinding:
       type: object
+      description: >-
+        A instance of EdgeBinding is a single KnowledgeGraph Edge mapping,
+        identified by the corresponding 'id' object key identifier of the
+        Edge within the Knowledge Graph. Instances of EdgeBinding may include
+        extra annotation (such annotation is not yet fully standardized).
       properties:
         id:
           type: string
-          description: A knowledge-graph edge id, i.e. the `id` of a KEdge
+          description: The key identifier of a specific KnowledgeGraph Edge.
       additionalProperties: true
       required:
         - id
+      additionalProperties: true
```

Comment:
Looks like this line duplicates line 263

Reply:
> Looks like this line duplicates line 263

Yeah...  @rtroper ... not sure how that slipped in. Fixed and committed the patch.